src/main/java/meteordevelopment/meteorclient/systems/hud/modules/CombatHud.java

```diff
@@ -41,6 +38,7 @@
 public class CombatHud extends HudElement {
     private static final Color GREEN = new Color(15, 255, 15);
     private static final Color RED = new Color(255, 15, 15);
+    private static final Color YELLOW = new Color(255, 150, 15);
     private static final Color BLACK = new Color(0, 0, 0, 255);
```

Comment:
Constants are in the `Color` class now; https://github.com/MeteorDevelopment/meteor-client/blob/master/src/main/java/meteordevelopment/meteorclient/utils/render/color/Color.java#L16-L28
Pretty recent change so no wonder that it's not changed

Reply:
Hmm I'll check that out